app/views/api/v2/ansible_variables/show.json.rabl

```diff
@@ -19,3 +19,5 @@ end
 node :override_values_count do |lk|
   lk.lookup_values.count
 end
+
+node :hidden_value, &:hidden_value?
```

Comment:
As I mentioned [here](https://github.com/theforeman/foreman_ansible/pull/716#issuecomment-2314902463), the `hidden_value` does not appear to be used in this query. Could you clarify how this change impacts the functionality? I might be missing the reason why it’s necessary.

Reply:
After looking into this further, I noticed that this change only affects the API call and does not impact the GraphQL query.